bookstore/client/nb_client.py

```diff
@@ -81,21 +82,21 @@ class NotebookSession:  # (NamedTuple):
     #     id: str #'68d9c58f-c57d-4133-8b41-5ec2731b268d',
     #     path: str #'Untitled38.ipynb',
     #     name: str #'',
-    #     type: str #'notebook',
+    #     kind: str #'notebook',
     #     kernel: KernelInfo
     #     notebook: dict # deprecated API {'path': 'Untitled38.ipynb', 'name': ''}}}
 
-    def __init__(self, *args, path, name, type, kernel, notebook, **kwargs):
+    def __init__(self, *args, path, name, kind, kernel, notebook, **kwargs):
```

Comment:
I would prefer that we don't change this as it would make it so that we cannot directly consume the contents of the API response from the `notebook`'s `/api/sessions/` in https://github.com/nteract/bookstore/pull/90/files#diff-7d77120266750554a7e864156f790108R139